e2e/landing.spec.ts

```diff
@@ -40,35 +36,78 @@ test.afterAll(async () => {
   await page.close();
 });
 
-test('Should render', async () => {
-  await expect(page).toHaveTitle(
-    'Learn to Code — For Free — Coding Courses for Busy People'
+test('The component Landing-top renders correctly', async () => {
+  const landingHeading1 = page.getByTestId('landing-big-heading-1');
+  await expect(landingHeading1).toHaveText('Learn to code — for free.');
+
+  const landingHeading2 = page.getByTestId('landing-big-heading-2');
+  await expect(landingHeading2).toHaveText('Build projects.');
+
+  const landingHeading3 = page.getByTestId('landing-big-heading-3');
+  await expect(landingHeading3).toHaveText('Earn certifications.');
+
+  const landingH2Heading = page.getByTestId('landing-h2-heading');
+  await expect(landingH2Heading).toHaveText(
+    'Since 2014, more than 40,000 freeCodeCamp.org graduates have gotten jobs at tech companies including:'
   );
-  const callToAction = page.getByTestId(landingPageElements.callToAction);
-  const callToActionHeader = page.locator('a .login-btn-text').nth(1);
-  await expect(callToActionHeader).toHaveText("Get started (it's free)");
-  await expect(callToAction).toHaveCount(4);
 });
 
-test('Has visible header and sub-header', async () => {
-  const heading = page.getByTestId(landingPageElements.heading);
-  await expect(heading).toContainText('Learn to code — for free.');
-  expect(await page.isVisible('text=Build projects.')).toBeTruthy();
-  expect(await page.isVisible('text=Earn certifications.')).toBeTruthy();
-  expect(
-    await page.isVisible(
-      'text=Since 2014, more than 40,000 freeCodeCamp.org ' +
-        'graduates have gotten jobs at tech companies including:'
-    )
-  ).toBeTruthy();
+test('Has 5 brand logos', async () => {
+  const logos = page.getByTestId('brand-logo-container');
+  expect(await logos.nth(0).isVisible());
+  expect(await logos.nth(1).isVisible());
+  expect(await logos.nth(2).isVisible());
+  expect(await logos.nth(3).isVisible());
+  expect(await logos.nth(4).isVisible());
```

Comment:
```suggestion
  const logos = page.getByTestId('brand-logo-container').locator('svg');
  await expect(logos).toHaveCount(5);
  for (const logo of await logos.all()) {
    await expect(logo).toBeVisible();
  }
```
This is a little more explicit (i.e. it checks there are 5 logos, not at least 5) and slightly more concise.